### PR TITLE
bootstrapping changes

### PR DESCRIPTION
**Pull request by gamerson**

### CODE REVIEW

**Comment by gamerson** [comment]
@bjhargrave @pkriens  finished up the changes to get the new build going and all the tests passing.  Do you guys have anything else you would like to see done before we merge?

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  - oraclejdk8
+
+before_install:
+  - unset _JAVA_OPTIONS
+
+install:
+  - ./mvnw --version
+
+script:
+  - sudo ./mvnw clean install
```
Comment (bjhargrave) [inline .travis.yml]: I would not do clean since the build starts in an empty workspace.

File: appveyor.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+environment:
+  matrix:
+    - JAVA_HOME: C:\Program Files\Java\jdk1.8.0
+
+install:
+  - mvnw.cmd --version
+
+build_script:
+  - mvnw.cmd clean install
```
Comment (bjhargrave) [inline appveyor.yml]: No clean here also.

File: appveyor.yml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+on_success:
+  - git status
```
Comment (bjhargrave) [inline appveyor.yml]: You don't need the git status here I think.

**Review by bjhargrave — CHANGES_REQUESTED** [review]
I see the tests don't work properly:

https://ci.appveyor.com/project/bndtools/jpmcli/build/9#L590

https://travis-ci.org/bndtools/jpmcli/builds/309594169#L1076

**Review by bjhargrave — APPROVED** [review]

**Pull request merged**